overlord/snapstate/autorefresh_gating.go

```diff
@@ -551,6 +551,18 @@ func createGateAutoRefreshHooks(st *state.State, affectedSnaps map[string]*affec
 	return ts
 }
 
+func conditionalAutoRefreshAffectedSnaps(t *state.Task) ([]string, error) {
+	var snaps map[string]*refreshCandidate
```

Comment:
as we are not looking at the content we can make the values *json.RawMessage perhaps

Reply:
Done.